packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-metadata.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import path from 'path';
+
+/*
+  METADATA GENERATION
+  Generates the /src/temp/metadata.json file which contains application 
+  configuration metadata that is used for Sitecore XM Cloud integration.
+*/
+generateMetadata();
+
+interface Metadata {
+  packages: { [key: string]: string };
+}
+
+function generateMetadata(): void {
+  const metadata: Metadata = { packages: {} };
+  const trackedScopes = ['@sitecore', '@sitecore-cloudsdk', '@sitecore-feaas', '@sitecore-jss'];
+  const dirs = fs.readdirSync('node_modules');
+
+  dirs.forEach(dir => {
+    if (trackedScopes.includes(dir)) {
+      const packages = fs.readdirSync(path.join('node_modules', dir));
```

Comment:
Does this line work for "@sitecore-jss" scope? There would be no node_modules directly underneath @sitecore-jss folder - it would be under @sirecore-jss/sitecore-jss-nextjs etc.

Reply:
yea, the same applies for the other scopes. The line you are refering to reads the packages under the scope, so in the above 'node_modules' is the app's folder. I double checked to be sure.